experience/index.js

```diff
@@ -1,11 +1,52 @@
+var Promise = require('sync-p')
 var createRegister = require('./createRegister')
 
 module.exports = function (meta) {
   var owner = meta && (meta.owner || meta.experimentId)
   if (!owner) {
     throw new Error('No owner specified')
   }
+
+  var register = createRegister(owner)
+  var release = null
+  var render = null
+  var React = null
+
   return {
-    register: createRegister(owner)
+    getReact: function () {
+      if (!React) {
+        throw new Error('React not available, you probably forgot to call `options.react.register()`')
```

Comment:
Again, we should be able to tell if they've done this

Reply:
I will update the wording on the error.